idaes/core/util/tests/test_model_diagnostics.py

```diff
@@ -1433,8 +1436,8 @@ def test_report_numerical_issues_jacobian(self):
         model.v2 = Var(initialize=0)
         model.v3 = Var(initialize=0)
 
-        model.c1 = Constraint(expr=model.v1 == model.v2)
-        model.c2 = Constraint(expr=model.v1 == 1e-8 * model.v3)
+        model.c1 = Constraint(expr=1e-2 * model.v1 == model.v2)
+        model.c2 = Constraint(expr=1e-2 * model.v1 == 1e-8 * model.v3)
         model.c3 = Constraint(expr=1e8 * model.v1 + 1e10 * model.v2 == 1e-6 * model.v3)
```

Comment:
Similarly, I updated coefficients on `v1` to give some buffer around the relative tolerance. The small coefficients on `v3` are also causing it to appear as "parallel to" to the other variables, which again raises the question of how to handle zero vectors.